recipes/cajun-jsonapi/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+find_package(cajun-jsonapi REQUIRED)
+
+add_executable(${PROJECT_NAME} main.cpp)
+target_link_libraries(${PROJECT_NAME} PRIVATE cajun::jsonapi)
```

A little cleaner approach to set the standard

```suggestion
target_link_libraries(${PROJECT_NAME} PRIVATE cajun::jsonapi)
set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 14)
```

---

```suggestion
target_link_libraries(${PROJECT_NAME} PRIVATE cajun-jsonapi::cajun-jsonapi)
```